hours/api.py

```diff
@@ -352,3 +377,12 @@ class DailyHoursViewSet(viewsets.ReadOnlyModelViewSet):
 
 
 register_view(DailyHoursViewSet, "daily_hours")
+
+
+class OrganizationViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Organization.objects.all()
+    serializer_class = OrganizationSerializer
+    filterset_fields = ["parent"]
+
+
+register_view(OrganizationViewSet, "organization")
```

Comment:
Do we really want to expose the organisation in the API?

Reply:
In my opinion, yes, since the organization data is public anyway and it might be useful to the user to know what the place of their organization is in the hierarchy? That dictates admin rights. Of course, if the UI doesn't want to use the data for figuring out user rights, then that is not needed at the moment.

> Looks ok, but nothing about "tprek organizations" here?

All the tprek is in the single line `django-orghierarchy==0.1.27` :P The rest is just supporting stuff for that.